src/components/customer/appsco-customers-actions.html

```diff
@@ -44,6 +73,22 @@
         </div>
 
         <div class="action flex-none">
+            <paper-button class="import-action"
```

Comment:
Missing imports for used components. paper-button, paer-listbox, paper-item, iron-icon, iron-icons (and social-icons). Check if I missed any.